src/utils/ssr/getExperimentInfo.js

```diff
@@ -13,9 +12,14 @@ const getExperimentInfo = async (context, store) => {
     return;
   }
 
+  const user = await Auth.currentAuthenticatedUser();
+  const jwt = user.getSignInUserSession().getIdToken().getJwtToken();
+
+  console.log(`beautiful jwt ${jwt}`);
+
   const url = `${req.protocol}://${req.get('host')}${req.originalUrl}`;
-  const apiUrl = getApiEndpoint(url);
```

Comment:
how is this going to work without calling the API endpoint?

Reply:
`getFromApiExpectOK` calls it internally, I refactored some of the code so it works more seamlessly with SSR stuff (adding a `uiUrl` parameter).

Reply:
ook, sounds good then!